geniesp/bpc_redcap_export_mapping.py

```diff
@@ -1067,10 +1066,11 @@ def run(self):
         del final_survivaldf["CANCER_INDEX"]
         # remove a row if patient ID is duplicated and PFS_I_ADV_STATUS is null or empty
         # tested on current survival data file and produces unique patient list
-        pfs_not_null_idx = ~final_survivaldf['PFS_I_ADV_STATUS'].isnull()
-        pfs_not_blank_idx = final_survivaldf['PFS_I_ADV_STATUS'] != ""
-        nondup_patients_idx = ~final_survivaldf['PATIENT_ID'].duplicated(keep=False)
-        final_survivaldf = final_survivaldf[(pfs_not_null_idx & pfs_not_blank_idx) | (nondup_patients_idx)]
+        if 'PFS_I_ADV_STATUS' in final_survivaldf.columns: 
```

Comment:
This is unfortunate - It's clear that there are different filtering strategies depending on the different cohorts.

Reply:
This may be because the data elements catalog hasn't been updated to appropriately reflect updates to the scope of release.  According to the scope of release, this variable should be included.